src/dug/core/async_search.py

```diff
@@ -221,9 +222,11 @@ async def search_concepts(self, query, offset=0, size=None, types=None,
         """
         Changed to a long boolean match query to optimize search results
         """
-        query_dict = self._build_concepts_query(query, **kwargs)
+        if "*" in query or "\"" in query or "+" in query or "-" in query:
```

Comment:
We should definitely document this in internal documents somewhere -- this is a valuable way of comparing `get_simple_search_query()` and `_build_concepts_query()` so we can see the advantages and disadvantages of each approach. I'm not sure if there's a good place to document this -- maybe open a new Dug ticket in Jira describing this and asking someone to compare results between the two approaches?

Reply:
https://renci.atlassian.net/browse/DUG-476